plugins/Ecommerce/VisitorDetails.php

```diff
@@ -165,19 +167,30 @@ protected function queryEcommerceConversionsForVisits($idVisits)
      */
     protected function queryEcommerceItemsForOrder($idVisit, $idOrder)
     {
+        $categorySelects = [];
+        $categoryJoins = [];
+        for ($i = 0; $i < self::CATEGORY_COUNT; ++$i) {
+            $suffix = $i === 0 ? '' : $i;
+            $column = $i === 0 ? 'idaction_category' : ('idaction_category' . ($i + 1));
+            $categorySelects[] = 'log_action_category' . $suffix . '.name as itemCategory' . $suffix;
+            $categoryJoins[] = 'LEFT JOIN ' . Common::prefixTable('log_action') . " AS log_action_category$suffix
```

Comment:
I reckon performance won't be a big problem because of adding so many more joins but may be still good to do (maybe we have many categories in the demo on staging?).

Reply:
If easily doable...

Reply:
You mean performance test on staging?

Reply:
Yeah I wonder if we should deploy the change on the server and check if it still loads as fast when eg loading 500 or 1000 ecommerce log entries compared to before? not sure we have many entries with categories though

Reply:
Could always add some more categories to the test data and load it on staging if not. Would take a couple days though.

Reply:
Nah not needed to add more categories. Not worth it I think